lldb/source/Commands/CommandObjectHealthcheck.cpp

```diff
@@ -60,8 +60,7 @@ bool CommandObjectHealthcheck::DoExecute(Args &args,
   // When in an interactive graphical session and not, for example,
   // running LLDB running over ssh, open the log file straight away in
   // the user's configured editor or the default Console.app otherwise.
-  if ((strcmp("lldb", getprogname()) == 0 ||
-       strcmp("lldb-rpc-server", getprogname()) == 0) &&
+  if (llvm::StringRef(getprogname()).starts_with("lldb") &&
```

Comment:
I'm having a hard time remembering why we were checking `argv[0]` here in the first place.

Reply:
Ah. The test suite, where it is `python`!